next.config.js

```diff
@@ -21,24 +23,21 @@ module.exports = (phase) => {
     },
   };
 
-  const redirects = () => {
-    return [
-      {
-        source: "/works",
-        destination: "/search",
-        permanent: true,
-      },
-    ];
-  };
-
   return {
     env,
+    output: "export",
```

Comment:
Wit this change, we no longer need `npm run build:static`. I think we can remove references to it in the package.json and the gh-pages.deploy.yml